src/qute/commands/commandConstants.ts

```diff
@@ -25,6 +25,16 @@ export namespace QuteClientCommandConstants {
    */
   export const COMMAND_CONFIGURATION_UPDATE = 'qute.command.configuration.update';
 
+  /**
+   * Toggle on extension for Qute validation setting.
+   */
+  export const TOGGLE_ON = '.toggle.on';
```

Comment:
remove this constant

Reply:
This was added based on your comment: https://github.com/redhat-developer/vscode-quarkus/pull/455#discussion_r803004245, would the full name be better here?

Reply:
Yes please